_includes/navbar.html

```diff
@@ -1,10 +1,12 @@
 <header class="nav-bar">
     <div class="nav-content">
-        <a href="{{ '/' | relative_url }}">
-            <div class="nav-logo">
-                <img src="./assets/img/logo.svg" />
-                <a class="nav-link" href="{{ site.url }}/cv/resume.pdf">Resume</a>                
-            </div>
-        </a>        
+        <div class="nav-logo">
+            <a href="{{ '/' | relative_url }}"><img src="./assets/img/logo.svg" /></a>    
```

Comment:
```suggestion
            <a href="{{ '/' | absolute_url }}"><img src="./assets/img/logo.svg" /></a>    
```

I think this was a mistake made upstream, since the URL passed to `relative_url` is already relative. It should be absolute.